apps/web-giddh/src/app/purchase/order-history/order-history.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit, Output, EventEmitter, ViewChild, ElementRef, TemplateRef } from '@angular/core';
+import {
+    BsDropdownDirective,
+    BsModalRef,
+    BsModalService,
+    ModalDirective,
+    ModalOptions,
+    TabsetComponent, 
+    PopoverDirective
+} from 'ngx-bootstrap'
+import { GeneralService } from 'apps/web-giddh/src/app/services/general.service';
+@Component({
+    selector: 'aside-order-history',
+    templateUrl: './order-history.component.html',
+    styleUrls: ['./order-history.component.scss']
+})
+export class OrderHistoryComponent implements OnInit{
+    public modelRef: BsModalRef;
```

Comment:
Add JS doc on all members and methods. Add return type in methods

Reply:
this is unused.